src/main/kotlin/com/tylerthrailkill/helpers/prettyprint/PrettyPrint.kt

```diff
@@ -152,6 +153,20 @@ private class PrettyPrinter(val tabSize: Int, val writeTo: Appendable, val wrapp
         write(']')
     }
 
+    private fun ppArray(obj: Array<*>, currentDepth: String) {
+        val increasedDepth = deepen(currentDepth)
+
+        // https://stackoverflow.com/questions/35938906/why-doesnt-kotlin-arrayt-implement-iterablet
+        val iterableList = obj.toList()
+
+        writeLine('[')
+        iterableList.ppContents(currentDepth, ",") {
+            write(increasedDepth)
+            ppAny(it, increasedDepth)
+        }
+        write(']')
+    }
+
```

Comment:
To reduce duplication, `ppArray` should just call `ppIterable`. And could you make a doc comment and put the link in that?

```kotlin

    /**
      * https://stackoverflow.com/questions/35938906/why-doesnt-kotlin-arrayt-implement-iterablet
    **/
    private fun ppArray(obj: Array<*>, currentDepth: String) {
        val iterableList = obj.toList()

		ppIterable(iterableList, currentDepth)
    }
```